assets/src/scripts/components/hydrograph/layout.js

```diff
@@ -49,7 +49,8 @@ export const layoutSelector = createSelector(
             windowWidth: windowWidth,
             margin: {
                 ...margin,
-                left: margin.left + approxLabelLength
+                left: margin.left + approxLabelLength,
```

Comment:
Is it possible to test this part of layoutSelector?

Reply:
Added.